content/ember-data/v3/default-serializer.md

```diff
@@ -17,21 +19,21 @@ These classes specified the following `defaultSerializer`
 - `RESTAdapter`: `-rest` (`@ember-data/serializer/rest`)
 - `JSONAPIAdapter`: `-json-api` (`@ember-data/serializer/json-api`)
 
-If a per-type adapter exists for the modelName triggering this deprecation, the easiest resolution is to add a per-type serializer.
-
-If the application adapter is triggering this deprecation, then an application serializer should be added.
-
-More information on adding these serializers can be found in "clearing these deprecations" below.
+### Clearing these deprecations
 
-### -default serializer fallback in store.serializerFor
+If all the adapters in your app are the same kind (such as JSONAPI, REST, or JSON), you should create an
+application serializer to match.
 
-Previously, when no type-specific serializer, application serializer, or adapter defaultSerializer had been defined by the app, the `-default` serializer would be used which defaulted to the `JSONSerializer`. This behavior is deprecated in favor of explicitly defining an application or type-specific serializer as described below.
+For example, if you use only JSONAPI adapters, creating the file below in `app/serializers/application.js`
+will resolve the deprecation:
 
-## clearing these deprecations
+```javascript
+// app/serializers/application.js
 
-More information about custom serializers can be found in the [Serializer API Docs](https://api.emberjs.com/ember-data/release/modules/@ember-data%2Fserializer) or on the [ember.js/guides](https://guides.emberjs.com/release/models/customizing-serializers/#toc_customizing-serializers)
+export { default } from '@ember-data/serializer/json-api';
```

Comment:
Were these module paths available in 3.12?

Reply:
@locks i think so 🤷‍♀ 
https://github.com/emberjs/data/blob/e0732481885abde4eb187d3a1573276374068a2f/packages/serializer/addon/json-api.js#L412

Reply:
https://guides.emberjs.com/v3.14.0/models/defining-models/
We seem to only have switched the defaults in 3.15 though. Should we align these with the default at the time of the deprecation?

Reply:
yeah makes sense